packages/@react-types/autocomplete/src/index.d.ts

```diff
@@ -66,6 +70,8 @@ export interface SpectrumSearchAutocompleteProps<T> extends SpectrumTextInputBas
    * @default true
    */
   shouldFlip?: boolean,
+  /** Width of the menu. */
```

Comment:
```suggestion
  /** Width of the menu. By default, matches width of the trigger. */
```

Reply:
@snowystinger Hey, appreciate your feedback!

By the way, since all three components (`Picker`, `ComboBox`, `SearchAutocomplete`) have this prop, do you want me to go ahead and apply the change to all three?

Reply:
yep, that'd be great